README.md

```diff
@@ -186,6 +191,18 @@ Path to Android NDK []: C:\tools\android-ndk
 Path to Java JDK [C:\Program Files\Java\jdk1.8.0_131]:
 ```
 
+#### Setup AppleTV
+- Run `lime rebuild hxcpp tvos`
+- Remove stable lime `haxelib remove lime`
+- Install lime from GitHub `git clone --recursive https://github.com/openfl/lime`
```

Comment:
In base have not tvos, webos, tizen and many other platforms. tvos have many errors, i think other platform too. Maybe it's reason for none provide by default. Anyway tvos need support, apple love it.